snap/snapenv/snapenv.go

```diff
@@ -106,8 +106,17 @@ func userEnv(info *snap.Info, home string) map[string]string {
 	return result
 }
 
-// envMap creates a map from the given environment string list, e.g. the
-// list returned from os.Environ()
+var unsafeEnv = map[string]bool{
+	"HOSTALIASES": true,
```

Comment:
If the user sets HOSTALIASES to a directory that is not readable due to the sandbox shoud this be still preserved?